README.md

```diff
@@ -0,0 +1,67 @@
+# The "Do All Stuff" (DAS) Telegram Bot #
+
+The DAS bot is a webhook based telegram bot written in Python 3.7 deployed on AWS Lambda. It is capable of fetching jokes, fun facts, as well as as multiple choice trivia questions and fetch insights from Google Trends (As along as those APIs' remain functional).
+
+## Usage ##
+Interact with the bot **[@thedasbot](https://t.me/thedasbot)** on Telegram. It supports both private and group chats.
+
+## Commands ##
+### Help ###
+Pulls up the list of available commands.
+
+**Syntax:** `/help`
+
+
+### Trivia ###
+Gets a random, interactable multiple choice question from [Open Trivia Database](https://opentdb.com/). 
+
+**Syntax:** `/trivia [ <category> | any | help ] [ <difficulty> | any ]`
+
+- `/trivia help` brings up the list of available categories and difficulties. Categories do not need to exactly match what's listed, as long it's enough to uniquely identify it. Leaving any arguments blank will default to `any`, which selects a random category/difficulty.
+
+**Examples**
+
+* `/trivia` - Question from random category and difficulty
+
+* `/trivia history` - Question from the "History" category at random difficulty
+
+* `/trivia anime easy` - Easy question from the "Entertainment: Japanese Anime & Manga" category
+
+* `/trivia any hard` - Hard question from a random category
```

Comment:
Can you include pictures of the bot from Telegram here? It would be nice to see the UI here on Github,